doc/classes/Object.xml

```diff
@@ -369,7 +369,8 @@
 			<argument index="1" name="default" type="Variant" default="null" />
 			<description>
 				Returns the object's metadata entry for the given [code]name[/code].
-				Throws error if the entry does not exist, unless [code]default[/code] is not [code]null[/code] (in which case the default value will be returned).
+				Throws error if the entry does not exist, unless [code]default[/code] is not [code]null[/code] (in which case the default value will be returned). See also [method has_meta], [method set_meta] and [method remove_meta].
+				[b]Note:[/b] Metadata whose [code]name[/code] starts with an underscore ([code]_[/code]) is considered editor-only. Editor-only metadata is not displayed in the inspector.
```

Comment:
```suggestion
				[b]Note:[/b] Metadata that has a [param name] starting with an underscore ([code]_[/code]) is considered editor-only. Editor-only metadata is not displayed in the inspector and should not be edited.
```
I think "whose" can only be used in reference to a living entity. Admittedly, changing this bit is kinda awkward, but I think this works? I'd also add a note that we do not recommend editing it (as I suggested, or phrased differently if you want).